lib/dw/dataset/column.js

```diff
@@ -225,6 +225,15 @@ export default function Column(name, rows, type) {
             return column;
         },
 
+        deleteRow(i) {
+            if (!arguments.length) return;
+            rows.splice(i, 1);
```

Comment:
I think we should remove the row from `origRows` as well, otherwise `values()` with the `unfiltered` argument would return a different row than when called without the `unfiltered` argument, which might be unexpected.